src/app.module.ts

```diff
@@ -62,6 +65,7 @@ import { PlayersOnline } from './players-online/entities/players-online.entity';
     AccountModule,
     AuthModule,
     PlayersOnlineModule,
+    NewsTickerModule,
```

Comment:
News Ticker cannot be a table into Game database, all those sites information has to be persisted into the 'f3sgames-api' database a.k.a "payment-db"